src/commcare_cloud/commands/ansible/service.py

```diff
@@ -209,8 +210,12 @@ def run_status_for_service_group(self, service_group, args, unknown_args):
                     args.inventory_group = self.get_inventory_group_for_service(service, args.service_group)
                     exit_code = RunShellCommand(self.parser).run(args, unknown_args)
                 if exit_code is not 0:
-                    # if any service status check didn't go smoothly exit right away
-                    return exit_code
+
```

Comment:
nit: a blank link here, probably not needed.
Also can name this as `failed_exit_codes` instead of `exit_codes`